map/src/menu/tracks/CloudTrackItem.jsx

```diff
@@ -100,26 +144,69 @@ export default function CloudTrackItem({ file, customIcon = null }) {
+                        className={styles.item}
+                        divider
+                        id={'se-cloud-track-' + trackName}
+                        onClick={() => setDisplayTrack(true)}
+                    >
+                        <ListItemIcon className={styles.icon}>
+                            <TrackIcon />
+                        </ListItemIcon>
+                        <ListItemText>
+                            <Typography variant="inherit" className={styles.groupName} noWrap>
                                 {trackName}
                             </Typography>
+                            <Typography variant="body2" className={styles.groupInfo} noWrap>
+                                {dist && `${dist} km · `}
+                                {time && `${time}`}
+                                {points && ` · ${points}`}
+                            </Typography>
                         </ListItemText>
-                        <Switch
-                            disabled={loadingTrack}
-                            onClick={(e) => e.stopPropagation()}
-                            checked={!!ctx.gpxFiles[file.name]?.url}
-                            onChange={(e) => !file.local && setDisplayTrack(e.target.checked)}
-                        />
+                        {visible ? (
+                            <Switch
+                                onClick={(e) => e.stopPropagation()}
+                                checked={!!ctx.gpxFiles[file.name]?.url}
+                                onChange={(e) => !file.local && setDisplayTrack(e.target.checked)}
+                            />
+                        ) : (
+                            <IconButton
+                                className={styles.icon}
+                                onMouseEnter={() => setHoverIconInfo(true)}
+                                onMouseLeave={() => setHoverIconInfo(false)}
+                                onClick={(e) => {
+                                    setOpenActions(true);
+                                    ctx.setOpenedPopper(anchorEl);
+                                    e.stopPropagation();
+                                }}
+                                ref={anchorEl}
+                            >
+                                {hoverIconInfo ? <MenuIconHover /> : <MenuIcon />}
+                            </IconButton>
+                        )}
                     </MenuItem>
                 </Tooltip>
+                <Popper
+                    sx={{ zIndex: 2000, mt: '-35px !important', ml: '-5px !important' }}
+                    open={openActions}
+                    anchorEl={anchorEl.current}
+                    disablePortal={true}
+                >
+                    <ClickAwayListener onClickAway={() => setOpenActions(false)}>
+                        <TrackActions track={file} setShowTrack={setShowTrack} />
+                    </ClickAwayListener>
```

Comment:
ClickAwayListener(s) should have more complicated onClickAway listener to work with "click-out" correctly (see PointContextMenu). Please fix and recheck all other ClickAwayListener(s) which were added recently